packaging/.gbs.conf

```diff
@@ -18,6 +18,9 @@ obs = obs.tizen
 repos = repo.unified, repo.base
 buildroot = ~/GBS-ROOT-Snapshot/
 
+# Install tizen.unified.conf at ~/ folder
+buildconf = ~/tizen.unified.conf
```

Comment:
I have changed the the existing "~/.gbs.conf" file. 
* before: repos = repo.aic, repo.unified, repo.base
* after: repos = repo.aic, repo.base, repo.unifed

Reply:
Starting the build test with PR https://github.com/nnsuite/nnstreamer/pull/1236.

Reply:
@myungjoo ,  Please update the PR with the below statement. 
* before: repos = repo.aic, repo.unified, repo.base
* after:    repos = repo.aic, repo.base, repo.unifed